src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor

```diff
@@ -66,12 +67,12 @@
 
             @if (navLinkItem.Url.HasValue())
             {
-                <a href="@(navLinkItem.Disabled ? "/" : navLinkItem.Url)"
+                <a href="@(navLinkItem.IsEnabled ? navLinkItem.Url : "/")"
+                   style="padding: 0px 20px 0px @(((navLinkItem.Depth * 14) + 27).ToString())px;"
```

Comment:
please remove the `px` unit where the value of the CSS property is equal to 0.

Reply:
done